tests/test_ssl.py

```diff
@@ -0,0 +1,259 @@
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                client_key_file=self.key_file,
+                client_cert_file=self.cert_file,
+                client_ca_file=self.invalid_file),
+            SslTestSubcase(
+                name="key_crt_ca_server_and_client_empty_crt",
+                expectSSLError=True,
+                server_key_file=self.key_file,
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                client_key_file=self.key_file,
+                client_cert_file=self.empty_file,
+                client_ca_file=self.ca_file),
+            SslTestSubcase(
+                name="key_crt_ca_server_and_client_empty_key",
+                expectSSLError=True,
+                server_key_file=self.key_file,
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                client_key_file=self.empty_file,
+                client_cert_file=self.cert_file,
+                client_ca_file=self.ca_file),
+            SslTestSubcase(
+                name="key_crt_ca_server_and_client_empty_ca",
+                expectSSLError=True,
+                server_key_file=self.key_file,
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                client_key_file=self.key_file,
+                client_cert_file=self.cert_file,
+                client_ca_file=self.empty_file),
+            SslTestSubcase(
+                name="key_crt_ca_ciphers_server_and_key_crt_ca_client",
+                server_key_file=self.key_file,
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                server_ciphers="ECDHE-RSA-AES256-GCM-SHA384",
+                client_key_file=self.key_file,
+                client_cert_file=self.cert_file,
+                client_ca_file=self.ca_file),
+            SslTestSubcase(
+                name="key_crt_ca_ciphers_server_and_client",
+                server_key_file=self.key_file,
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                server_ciphers="ECDHE-RSA-AES256-GCM-SHA384",
+                client_key_file=self.key_file,
+                client_cert_file=self.cert_file,
+                client_ca_file=self.ca_file,
+                client_ciphers="ECDHE-RSA-AES256-GCM-SHA384"),
+            SslTestSubcase(
+                name="non_equal_ciphers",
+                expectSSLError=True,
+                server_key_file=self.key_file,
+                server_cert_file=self.cert_file,
+                server_ca_file=self.ca_file,
+                server_ciphers="ECDHE-RSA-AES256-GCM-SHA384",
+                client_key_file=self.key_file,
+                client_cert_file=self.cert_file,
+                client_ca_file=self.ca_file,
+                client_ciphers="TLS_AES_128_GCM_SHA256"),
```

Comment:
Does Tarantool support multiple ciphers? maybe a test specifying more that one cipher would be a great addition.

Reply:
Added